src/frontend/src/optimizer/plan_node/logical_update.rs

```diff
@@ -43,14 +39,8 @@ pub struct LogicalUpdate {
 
 impl From<generic::Update<PlanRef>> for LogicalUpdate {
     fn from(core: generic::Update<PlanRef>) -> Self {
-        let ctx = core.input.ctx();
-        let schema = if core.returning {
-            core.input.schema().clone()
-        } else {
-            Schema::new(vec![Field::unnamed(DataType::Int64)])
-        };
```

Comment:
The schema is changed?

Reply:
The same logic has been implemented in generic::Update